idaes_examples/notebooks/docs/unit_models/reactors/egprod_ideal.py

```diff
@@ -225,6 +224,6 @@
     "state_bounds": {"flow_mol_phase_comp": (0, 100, 1000,
                                              pyunits.mol/pyunits.s),
                      "temperature": (273.15, 298.15, 450, pyunits.K),
-                     "pressure": (5e4, 1e5, 1e6, pyunits.Pa)},
+                     "pressure": (1e3, 1e5, 1e6, pyunits.Pa)},
```

Comment:
This is a very low bound for P. why was it necessary to change it?

Reply:
The `eg_h2o_ideal` package had a value of `1e3` for the lower bound, and I changed the `egprod_ideal` file to match. I can change the lower bounds to `5e4` in both files.

Reply:
What are the limits on the correlations that are being used (if any)? 1e3 is very low for pressure (requires very specialised equipment to achieve), so I would be inclined to go with 5e4.

Reply:
There are no pressure limits for the liquid properties, only temperature limits. I'll go with 5e4 per your suggestion.